Log-Sources/Tausight/src/manifest.yaml

```
@@ -0,0 +1,11 @@
+author:
+  name: tausight
```

Please provide support email address
```suggestion
  name: Tausight
  email: support@tausight.com
```